### PR TITLE
Beta report

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 9/09_00_ARC09_2.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ I still have plenty of things I need to think about,
 [sha:38021f55a71778179e0b145e6cd8f65b754b0af5]{
 -- Page 10004, Offset 21383.
 -- 時刻は７時半。
-The time is seven-thirty.
+The time is 7:30.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 9/09_00_ARC09_2.txt:15]: It looks like we've been pretty inconsistent, but our editing doc says digits for anything that isn't on the hour 
https://github.com/Tsukihimates/Tsukihime-Translation/blob/main/Editing_Guidelines.md#time-format

File: script/Ciel/Day 8/08_01_CIEL08_1B.txt [hunk 2]
```diff
@@ -178,7 +178,7 @@ I think simple relationships are useful in their own way.
 [sha:b42136e15a7110c594f0275e63c1eace1d942eb4]{
 -- Page 46, Offset 18377.
 -- 色んなところに行けるようになって、いろんな人と知り合えるようになった以上、人間関係も最適化させないと潰れちまう……ってのは友人からの受け売りだけど」
-Once you've been all sorts of places and met with all sorts of people,
+Once you've been to all sorts of places and met with all sorts of people,
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_1B.txt:181]: kinda went back and forth on this one but i think with the "_with_ all sorts" it works.

Comment (rschlaikjer) [reply script/Ciel/Day 8/08_01_CIEL08_1B.txt:181]: Yeah I didn't love this one either tbh, if it was also weird to you I'd just revert

Comment (Avuxo) [reply script/Ciel/Day 8/08_01_CIEL08_1B.txt:181]: i'm alright with reverting

**Review by yk1512 — APPROVED** [review]

**Review by Petrikow — APPROVED** [review]

**Pull request merged**